apps/frontend/src/app/Data/Characters/Yoimiya/index.tsx

```diff
@@ -102,12 +102,17 @@ const normalEntries = dm.normal.hitArr.map((arr, i) => [
   dmgNode("atk", arr, "normal", { hit: { ele: compareEq(condSkill, "skill", elementKey, "physical") } }, normal_dmgMult)
 ])
 
-// This might just need to be a single dmgNode of her kindling arrow, with proper scaling applied.
 const kindlingEntries = dm.normal.hitArr.map((arr, i) => [i, greaterEq(input.constellation, 6,
   equal(condSkill, "skill",
     customDmgNode(
       prod(
-        subscript(input.total.autoIndex, arr, { unit: "%" }),
+        subscript(
+          input.total.autoIndex,
+          i === 0 || i === 3
```

Comment:
(nit) can you extract this out in a separate line `cont skillHitArr = ...` with comment